### DISCUSSION TITLE
Cloud building electron for MacOS

### DISCUSSION THREAD

**Discussion by customautosys** [quasarframework/quasar#8776] [CLI - Electron mode]
Hi, I need to build my release dmg on the cloud because my Mac is currently borked. I've tried Travis CI and it failed miserably (it did build the dmg, but it couldn't upload it to sftp and when I tried to install sshpass with homebrew so that I could use it to connect to my sftp server, it caused the build to timeout and get automatically cancelled when installing sshpass). Do you know of any alternatives (preferably, I would like to be able to download the built file after it is built rather than having to write commands in a script to upload the built file somewhere else as that would mean that I have to rebuild the whole thing just because my upload command failed)?

**Comment by hawkeye64** [comment]
@siauderman If you have another computer available, you could set up a virtual environment: https://hackintosh.com/

**Reply by customautosys** [reply]
Hmm, but it still needs another Mac that supports Big Sur for me to get my hands on a copy of Big Sur.

Anyway, I decided to send the Mac for repair.

**Reply by ontwikkelfabriek** [reply]
@siauderman 
You can run MacOs in a VM as a guest on a Windows host machine with  VMWARE or VirtualBox.

**Reply by ontwikkelfabriek** [reply]
> @hawkeye64  @siauderman If you have another computer available, you could set up a virtual environment: https://hackintosh.com/

A hackintosh will run MacOS bare metal , there's nothing virtual about a hackintosh.
